magpie/api/management/user/user_utils.py

```diff
@@ -858,8 +854,8 @@ def get_user_service_resources_permissions_dict(user, service, request,
 
 
 def check_user_info(user_name=None, email=None, password=None, group_name=None,  # required unless disabled explicitly
-                    check_name=True, check_email=True, check_password=True, check_group=True):
-    # type: (Str, Str, Str, Str, bool, bool, bool, bool) -> None
+                    check_name=True, check_email=True, check_password=True, check_group=True, check_anonymous=True):
```

Comment:
Could the parameter be adjusted to `check_not_anonymous` to improve readability?
Looking at the code invoking this function, `check_anonymous=True` gives the impression that it should "*check the value matches anonymous*", while the opposite check is actually performed.